flopy/discretization/structuredgrid.py

```diff
@@ -988,8 +988,8 @@ def from_gridspec(cls, gridspec_file, lenuni=0):
         grd = cls(np.array(delc), np.array(delr), lenuni=lenuni)
         xll = grd._xul_to_xll(xul)
         yll = grd._yul_to_yll(yul)
-        cls.set_coord_info(xoff=xll, yoff=yll, angrot=rot)
-        return cls
+        grd.set_coord_info(xoff=xll, yoff=yll, angrot=rot)
+        return grd
```

Comment:
is it ok to return a reconstructed grid object (i.e. not modifying modelgrid inplace)?

Reply:
Modified as per @jlarsen-usgs suggestion. Good guess @langevin-usgs, not too desperate to dive down that rabbit hole! I am away from the computer for a short while now but can take a look at a test when I get back. Is there still appetite to continue to support gridspec files going forward?